callforce/.github
# Integrate GitHub Actions

### 🏗 Changes

* Adds `organization-labels-action` form standardizing labels across the org
* Adds `standardize labels` workflow
* Adds `lint`, `format`, and `test` workflows
* Add `maintenance` and `chore` org labels

### 📋 Testing Steps

1. Go to relevant commits on this branch
2. Verify that status checks show
3. Click on status checks to see GitHub Actions that have run

### :octocat: Related GitHub Issues

* [callforce/management#12](https://github.com/callforce/management/issues/12)

### 📝 Notes

_None_
